src/main/java/org/vaadin/easyuploads/client/ui/VMultiUpload.java

```diff
@@ -292,7 +296,7 @@ private void submit() {
             } else if (!AcceptUtil.accepted(file.getName(), file.getType(),
                     accepted)) {
                 noMatches.add(file);
-            } else if (maxFileCount != null && filedetails.size() >= maxFileCount) {
+            } else if (maxFileCount != null && filedetails.size() > maxFileCount) {

```

Comment:
Without this change I can't click to open the browser file selection dialog and successfully choose only one file.  I filed this as bug # 25 a while back.  Without this you get a message like "0 files selected, but the maximum allowed is 1."